diagnostics/model_accuracy/verify_inference_model.py

```diff
@@ -154,7 +131,10 @@ def check_accuracy(model, p, tolerance=0.001):
     #TODO(Sindhu): Run multiple or ALL models at once and compare accuracy.
 
     try:
-        model_name, p = run_inference(args.model_name, models_dir)
-        check_accuracy(model_name, p)
+        model_name, p = run_inference(args.model_name, models_dir,args.json_file_name)
+        check_accuracy(model_name, p,args.json_file_name)
+        sys.exit(0)
```

Comment:
sys.exit(0) should only occur if check_accuracy() has found that all models pass.  check_accuracy() returns True or False based on whether the models passed or failed.  Therefore, the code here should be:

if check_accuracy(model_name, p,args.json_file_name) :  sys.exit(0)
else:  sys.exit(1)

Reply:
@crlishka Thanks Chris. Changed.